Results.html

```diff
@@ -21,14 +22,14 @@
     <div class = "nav">
         <ul>
             <li> <a href="index.html"> About </a> </li>
-            <li> <a href="Results.html" class="selected"> Results </a> </li>
+            <li> <a href="results.html" class="selected"> Results </a> </li>
             <li> <a href="map.html"> Find the Coffee </a></li>
         </ul>
     </div>
 </header>
 <section class="roaster-winner">
```

Comment:
I agree, my approach to the css changed as I worked through this project. When I first started I styled every element alone before I realized I could make a formatting template and apply it to multiple items. I'll go through and give things more generic classes that can be used repeatedly!